cloud-arm/src/main/java/com/sequenceiq/cloudbreak/cloud/arm/ArmCredentialConnector.java

```diff
@@ -10,6 +10,10 @@
 @Service
 public class ArmCredentialConnector implements CredentialConnector {
 
+    @Override
```

Comment:
the credential verification code from com.sequenceiq.cloudbreak.service.credential.azurerm.AzureRmCredentialHandler#init() can be moved here, and com.sequenceiq.cloudbreak.service.credential.azurerm.AzureRmCredentialHandler can be removed.
With this change all the active cloud provider implementations will be under spi.